src/lib.rs

```diff
@@ -1,6 +1,4 @@
 #![doc(html_root_url = "https://hyperium.github.io/hyper/hyper/index.html")]
-#![deny(missing_docs)]
-#![cfg_attr(test, deny(warnings))]
```

Comment:
Hehe :)

As I mentioned in the opening comment, the PR is definitely not ready to be merged just yet.

I removed this since I wanted to be able to experiment with changes without being slowed down by having to put docs everywhere and get rid of warnings.

If we want to go forward with polishing this up, then of course all those issues will be solved and the commit that removed these two lines rebased out of the tree.